SoftLayer/CLI/hardware/notifications.py

```diff
@@ -18,12 +18,13 @@ def cli(env, identifier):
 
     notifications = hardware.get_notifications(identifier)
 
-    table = formatting.KeyValueTable(['Domain', 'Hostmane', 'Username', 'Email', 'FirstName', 'Lastname'])
+    table = formatting.KeyValueTable(['Id', 'Domain', 'Hostmane', 'Username', 'Email', 'FirstName', 'Lastname'])
```

Comment:
The comment https://github.com/softlayer/softlayer-python/issues/1772#issuecomment-1281464119 said also add `id` column in `vs notifications`, this will be resolved in other issue?